src/Nethermind/Nethermind.Core.Test/BlockValidatorTests.cs

```diff
@@ -35,10 +35,10 @@ public void When_more_uncles_than_allowed_returns_false()
             ISpecProvider specProvider = new CustomSpecProvider((0, releaseSpec));
 
             BlockValidator blockValidator = new BlockValidator(txValidator, AlwaysValidHeaderValidator.Instance, AlwaysValidOmmersValidator.Instance, specProvider, LimboLogs.Instance);
-            bool noiseRemoved = blockValidator.ValidateSuggestedBlock(Build.A.Block.TestObject);
+            bool noiseRemoved = blockValidator.ValidateSuggestedBlock(Builders.Build.A.Block.TestObject);
             Assert.True(noiseRemoved);
             
-            bool result = blockValidator.ValidateSuggestedBlock(Build.A.Block.WithOmmers(Build.A.BlockHeader.TestObject).TestObject);
+            bool result = blockValidator.ValidateSuggestedBlock(Builders.Build.A.Block.WithOmmers(Builders.Build.A.BlockHeader.TestObject).TestObject);
```

Comment:
We now have Prepare.A/Build.A/Builders.Build.A - what are the differences?

Reply:
it should be always Build A, i remiced prepare.a